libgalois/include/katana/GraphTopology.h

```diff
@@ -783,12 +790,18 @@ class KATANA_EXPORT ProjectedTopology : public GraphTopologyTypes {
   NUMAArray<Node> projected_to_original_nodes_mapping_;
   NUMAArray<Edge> original_to_projected_edges_mapping_;
   NUMAArray<Edge> projected_to_original_edges_mapping_;
+  NUMAArray<uint8_t> node_bitmask_;
+  NUMAArray<uint8_t> edge_bitmask_;
```

Comment:
Can we avoid storing this and generating this on-demand (when requested) from the mappings above? 

On that note, taking dependent arguments in the constructor is a risk. What if these variables are passed such that they are not agree-ing with each other?

If we need to store them, I'd suggest storing `DynamicBitset` and generating `NUMAAray<uint8_t>` on-demand (before calling Arrow) in the caller code (not in this call).

Reply:
the bitmask I thought were required to store, to even create the node properties.